src/cases/models/role-exclusions/role-exclusion.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface RoleExclusion {
+    type: string;
+    name: string;
+    userType: string;
+    Notes: string;
+    added: Date;
```

Comment:
More for my information - where is added being used?

Reply:
yes please

Reply:
What does "yes please" mean? Your comment makes no sense.

Reply:
added is being used inside the component - RoleExclusionsComponent @paulhowes-HMCTS

Reply:
Discussed with Uday. 'added' is the name of a field on the 'Display exclusions' section as per ticket.